### PR TITLE
Update: Site Responsiveness and minor changes

### PR DESCRIPTION
**Pull request by hussiantabha**

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -115,27 +115,38 @@ a{
     left: 0px;
     top: 8%;
     display: flex;
-    width: 20rem;
+    width: 17rem;
     flex-direction: column;
     justify-content: flex-start;
```
Comment (JantuDeb) [inline app.css]: This is redundant default value is `flex-start`.

**Review by JantuDeb — APPROVED** [review]
Suggested a changes.

File: components/card.css [hunk 2]
```diff
@@ -1,10 +1,8 @@
 .card-container{
     display: flex;
     flex-direction: column;
-    justify-content: space-evenly;
+    justify-content: center;
     align-items: center;
-    row-gap: 5rem;
-    flex-wrap: wrap;
     margin-bottom: 10rem;
```
Comment (mariyasada) [inline components/card.css:6]: use shorthand property flex-flow: column wrap

**Review by mariyasada — APPROVED** [review]

File: components/card.css (hunk 2 above)
Comment (hussiantabha) [reply components/card.css:6]: Done

**Pull request merged**